libensemble/tests/regression_tests/test_evaluate_existing_sample.py

```diff
@@ -16,48 +16,33 @@
 
 import numpy as np
 
-from libensemble.alloc_funcs.give_pregenerated_work import give_pregenerated_sim_work as alloc_f
-
 # Import libEnsemble items for this test
-from libensemble.libE import libE
+from libensemble import Ensemble
+from libensemble.alloc_funcs.give_pregenerated_work import give_pregenerated_sim_work as alloc_f
 from libensemble.sim_funcs.borehole import borehole as sim_f
 from libensemble.sim_funcs.borehole import gen_borehole_input
-from libensemble.tools import parse_args, save_libE_output
+from libensemble.specs import AllocSpecs, ExitCriteria, SimSpecs
 
 # Main block is necessary only when using local comms with spawn start method (default on macOS and Windows).
 if __name__ == "__main__":
-    nworkers, is_manager, libE_specs, _ = parse_args()
-
-    sim_specs = {
-        "sim_f": sim_f,
-        "in": ["x"],
-        "out": [("f", float)],
-    }
-
-    gen_specs = {}
 
     n_samp = 1000
-    n = 8
-
     H0 = np.zeros(n_samp, dtype=[("x", float, 8), ("sim_id", int), ("sim_started", bool)])
-
     np.random.seed(0)
     H0["x"] = gen_borehole_input(n_samp)
     H0["sim_id"] = range(n_samp)
     H0["sim_started"] = False
 
-    alloc_specs = {"alloc_f": alloc_f}
-
-    exit_criteria = {"sim_max": len(H0)}
-
-    # Perform the run
-    H, persis_info, flag = libE(
-        sim_specs, gen_specs, exit_criteria, alloc_specs=alloc_specs, libE_specs=libE_specs, H0=H0
-    )
-
-    if is_manager:
-        assert len(H) == len(H0)
-        assert np.array_equal(H0["x"], H["x"])
-        assert np.all(H["sim_ended"])
+    sampling = Ensemble()
+    sampling.H0 = H0
```

Comment:
I wonder if we should stick to function calls, matter of style. Or should we make this an option to Ensemble()?

Reply:
` sampling = Ensemble(H0=H0)`

Reply:
Both ways of setting H0 (and other attributes) are valid:

`sampling = Ensemble(H0=H0)`
or:
```
sampling = Ensemble()
sampling.H0 = H0
```